shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -67,7 +67,7 @@
 import org.robolectric.shadows.ShadowDisplay;
 
 @Config(manifest = Config.NONE)
-@RunWith(RobolectricTestRunner.class)
+@RunWith(AndroidJUnit4.class)
```

Comment:
`FlutterViewTest` uses `Shadow` APIs of `Robolectric`, it's better to keep it using `RobolectricTestRunner`.

Reply:
These specific unit tests are passing in CI. what issue did you find?  Robolectric's readme defaults to using AndroidJUnit4 https://github.com/robolectric/robolectric#usage

Reply:
This example doesn't use any shadow API of Robolectric. The tests use `AndroidJUnit4` also can run on real Emulator/device, and shared with [sharedTest pattern](http://robolectric.org/blog/2021/10/06/sharedTest/). If those tests will not run on real Emulator, it's fine to use `AndroidJUnit4`. If yes, we should avoid use shadow APIs for tests use `AndroidJUnit4`, or keep it to use `RobolectricTestRunner` for shadow APIs usage. I know there are some tests still use `ShadowDisplay` to control display rotation for tests.

Reply:
Oh, it's my fault. It uses `Robolectric` API. It's better to update it with `ActivityScenario` to lunch Activity, instead of current `Robolectric.setupActivity` in example. I will send a PR for it at Robolectric to discuss it.